app/components/tweet-type/component.js

```diff
@@ -1,6 +1,13 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
+  didInsertElement: function(){
+    this.$(".type").click(function(){
+      $(".type").removeClass("active");
```

Comment:
This is not Ember way of doing this. Ember way would be to use state to drive these property updates. You want the property on the component to change when a button is active and for component to update itself automatically without manually managing DOM updates.